build/gulp/plugins/webpack-open-browser.ts

```diff
@@ -0,0 +1,25 @@
+import open from 'open'
+
+type OpenBrowserPluginOptions = {
+  host: string
+  port: number
+}
+
+export default class OpenBrowserPlugin {
+  opened: boolean
+  options: OpenBrowserPluginOptions
+
+  constructor(options: OpenBrowserPluginOptions) {
+    this.options = options
+  }
+
+  apply(compiler) {
+    compiler.hooks.done.tap('Hello World Plugin', () => {
+      if (!this.opened) {
+        this.opened = true
+
+        open(`http://${this.options.host}:${this.options.port}`).catch(() => {})
```

Comment:
This is really annoying as it opens up a new doc tab even when it is already open.  I think the user should just open their own doc site when they are ready.

If the majority of the team wants this, let's duplicate what CRA does.  It only opens the tab if necessary otherwise it focuses the already open tab.

https://github.com/facebook/create-react-app/blob/2c6dd45cf4cb81f3cbd2e4073713647ebd033bbf/packages/react-dev-utils/openBrowser.js

Reply:
Switched this one to use `react-dev-utils/openBrowser` 👍